aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPluginKt.kt

```diff
@@ -24,22 +25,24 @@ import org.json.JSONObject
 /**
```

Comment:
should javadoc be directly above the class definition instead?

Reply:
I have not seen complaint from lint regarding it.